src/App.js

```diff
@@ -20,31 +18,37 @@ class App extends Component {
 
   componentDidMount = () => {
     fetchAllMovies()
-      .then((data) => this.setState({movies: data.movies, isLoading: false}))
+      .then(data => this.setState({ movies: data.movies, isLoading: false }))
   }
 
 
   setSelectedMovie = (id) => {
-    this.setState({selectedMovie: id})
+    this.setState({ selectedMovie: id });
   }
 
   toggleModal = (bool) => {
-    this.setState({isModalOpen: bool})
+    this.setState({ isModalOpen: bool });
   }
 
-
   render = () => {
     return (
       <div className="App">
         <Header />
```

Comment:
We may want to think about having the Header inside of our conditional rendering on line 38 and creating a Loading component that we can add to line 37 instead of just a Loading div.